src/promptflow/setup.py

```diff
@@ -64,6 +64,7 @@
             "promptflow-core[executor-service]",  # used to build web executor server
         ],
     },
+    scripts=["pf.autocomplete.sh"],
```

Comment:
@YingChen1996  Shall we inlcude this in devit setup?

Reply:
Please include in devkit setup, by the way, pyproject.toml doesn't have a perfect way for this script.
Please insert something into rc file, just like what conda did.

Reply:
I think no need put it in setup.  

Maybe put pf.autocomplete.sh in the folder src/promptflow-devkit/promptflow is enough since it's for pf command and add  `source <promptflow_package_install_root>/pf.completion.sh` in doc.

Reply:
Hello everyone, I am sorry I am a little lost since python is not one of my strongs. Where should I put the .sh file and in what setup.py file should I reference it ?

Also how do I include it in the rc file ? Conda seems to update $PATH but I do not see how to execute my pf.autocomplete.sh on env activate

Reply:
@sashokbg Do you mind us make some changes to this PR? 

I think the major function is ready in this PR. And we can tweak some minor details and get this merged. 
Please let us know if this works for you.

Reply:
Hello @wangchao1230 yes no problem at all. How would you like to proceed, should I stop pushing to this branch ?